tests/FertiScan/analysis_returned.json

```diff
@@ -0,0 +1,213 @@
+{
+    "company": {
```

Comment:
I suggest adding a `verified` field and setting it to false in the json returned from `new_inspection`. This will be used in the `update_inspection` function to create or update fertilizer records. It will help maintain a single json model between `new_inspection` and `update_inspection`.